### ISSUE TITLE
automatic version rewriting code busts package names in version objects.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3090]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#3090|
|Type|defect|
|Created|2008-03-18 00:32:35Z|
|Branch|https://github.com/twisted/twisted/tree/version-name-3090|

If you run Project.updateVersion, it writes a _version.py with a package name of "__builtins__" instead of the actual package name.

<details><summary>Searchable metadata</summary>

```
trac-id__3090 3090
type__defect defect
reporter__radix radix
priority__normal normal
milestone__twisted_8_0 twisted-8.0
branch__branches_version_name_3090 branches/version-name-3090
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1205800355000000 1205800355000000
changetime__1205804594000000 1205804594000000
version__None None
owner__radix radix

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_1"></a>|@radix commented|
|-|-|

(In [22923]) Branching to 'version-name-3090'

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)<a name="note_2"></a>|@radix set owner to @exarkun|
|-|-|

It wasn't a code problem after all, just the fact that the existing _version.py files were using dumb dynamic expressions instead of regular strings. I've changed this in a branch and now the version-updating code replaces it without inserting bizarre __builtin_ things.

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_3"></a>|@wsanchez commented|
|-|-|

reviewed

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)|@wsanchez set owner to @radix|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set status to `closed`|
|-|-|

(In [22931]) Merge version-name-3090

Author: radix
Reviewer: wsanchez
Commiter: therve
Joker: pahan
Fixes [#3090](https://github.com/twisted/twisted/issues/3090)

Fixes original content of _version file so that updateTwistedVersionInformation
works correctly.
